using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -295,25 +295,223 @@ To set up local notifications in Android, update the AndroidManifest.xml file wi
+            public void onComplete(@NonNull Task<InstanceIdResult> task) {
+                if (!task.isSuccessful()) {
+                    return;
+                }
+                // Get new Instance ID token
+                String registrationID = task.getResult().getToken();
+                // Log and toast
+                System.out.println("Received new registration token: " + registrationID);
+                // invoke the API to send the push identifier to the Identity Service
+                MobileCore.setPushIdentifier(registrationID);
+            }
+});
+```
+
 {% endtab %}
 
 {% tab title="iOS" %}
 {% hint style="warning" %}
 iOS simulators do not support push messaging.
 {% endhint %}
 
-To obtain the registration ID/token, see [Configuring Remote Notification Support](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/HandlingRemoteNotifications.html#//apple_ref/doc/uid/TP40008194-CH6-SW1).
+#### Objective-C
+
+#### Example
+
+```objectivec
+- (void) application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
+  // Set the deviceToken that the APNS has assigned to the device
+  [ACPCore setPushIdentifier:deviceToken];
+  //...
+}
+```
+
+#### Swift
+
+#### Example
+
+```swift
+func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
+  // Set the deviceToken that the APNS has assigned to the device
+  ACPCore.setPushIdentifier(deviceToken)
+  //...
+}
+```
+
 {% endtab %}
 
 {% tab title="React Native" %}
-Follow instructions in the Android/iOS tabs to set up platform-specific push configuration and use the following API in your React Native project:
+
+Prior to using the following API in your React Native project, complete the steps in the **Android** and **iOS** tabs to set up platform-specific push configuration.
+
+#### Example
+
+```javascript
+ACPCore.setPushIdentifier("pushID");
+```
+
+{% endtab %}
+{% endtabs %}
+
+## Tracking local and push notification message interactions
+
+User interactions with local or push notifications can be tracked by invoking the `collectMessageInfo` API. After the API is invoked, a network request is made to Campaign that contains the message interaction event.
+
+{% hint style="warning" %}
+The code samples below are provided as examples on how to correctly invoke the `collectMessageInfo` API. The Campaign documents on local and push notification tracking are the recommended source for the proper implementation of local and push notification message tracking. The Campaign document regarding local notification tracking can be seen at [Implementing local notification tracking](https://helpx.adobe.com/campaign/kb/local-notification-tracking.html#Description) and the document regarding push notification tracking can be see at [Push Tracking](https://helpx.adobe.com/campaign/kb/push-tracking.html).
```

Comment:
Please reword the 2nd sentence as follows:
"The Campaign document about local notification tracking is [Implementing local notification tracking](https://helpx.adobe.com/campaign/kb/local-notification-tracking.html#Description), and the Campaign document about push notification tracking is [Push Tracking](https://helpx.adobe.com/campaign/kb/push-tracking.html).